WordPress/Classes/Models/Notifications/Notification.h

```diff
@@ -64,47 +64,47 @@ typedef NS_ENUM(NSInteger, NoteBlockGroupType)
-@property (nonatomic, strong,  readonly) NotificationBlockGroup *headerBlockGroup;
-@property (nonatomic, strong,  readonly) NSArray                *bodyBlockGroups;   // Array of NotificationBlockGroup objects
-@property (nonatomic, assign,  readonly) NSNumber               *metaSiteID;
-@property (nonatomic, assign,  readonly) NSNumber               *metaPostID;
-@property (nonatomic, strong,  readonly) NSNumber               *metaCommentID;
-@property (nonatomic, strong,  readonly) NSNumber               *metaReplyID;
-@property (nonatomic, strong,  readonly) NSURL                  *iconURL;
-@property (nonatomic, strong,  readonly) NSDate                 *timestampAsDate;
-
-@property (nonatomic, assign,  readonly) BOOL                   isMatcher;
-@property (nonatomic, assign,  readonly) BOOL                   isComment;
-@property (nonatomic, assign,  readonly) BOOL                   isPost;
-@property (nonatomic, assign,  readonly) BOOL                   isFollow;
-@property (nonatomic, assign,  readonly) BOOL                   isLike;
-@property (nonatomic, assign,  readonly) BOOL                   isCommentLike;
-@property (nonatomic, assign,  readonly) BOOL                   isBadge;
-@property (nonatomic, assign,  readonly) BOOL                   hasReply;
+@property (nonatomic, strong, nullable, readonly) NotificationBlockGroup            *subjectBlockGroup;
+@property (nonatomic, strong, nullable, readonly) NotificationBlockGroup            *headerBlockGroup;
+@property (nonatomic, strong,  nonnull, readonly) NSArray<NotificationBlockGroup *> *bodyBlockGroups;
+@property (nonatomic, assign, nullable, readonly) NSNumber                          *metaSiteID;
+@property (nonatomic, assign, nullable, readonly) NSNumber                          *metaPostID;
+@property (nonatomic, strong, nullable, readonly) NSNumber                          *metaCommentID;
+@property (nonatomic, strong, nullable, readonly) NSNumber                          *metaReplyID;
+@property (nonatomic, strong, nullable, readonly) NSURL                             *iconURL;
+@property (nonatomic, strong,  nonnull, readonly) NSDate                            *timestampAsDate;
+
+@property (nonatomic, assign, readonly) BOOL                                        isMatcher;
+@property (nonatomic, assign, readonly) BOOL                                        isComment;
+@property (nonatomic, assign, readonly) BOOL                                        isPost;
+@property (nonatomic, assign, readonly) BOOL                                        isFollow;
+@property (nonatomic, assign, readonly) BOOL                                        isLike;
+@property (nonatomic, assign, readonly) BOOL                                        isCommentLike;
+@property (nonatomic, assign, readonly) BOOL                                        isBadge;
+@property (nonatomic, assign, readonly) BOOL                                        hasReply;
```

Comment:
I just died a little seeing this diff block #foreverunaligned

(kidding... kind of)